packages/teams-js/test/public/liveShare.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { LiveShareHost } from '../../src/internal/liveShareHost';
+import { liveShare } from '../../src/public/liveShare';
+
+describe('MicrosoftTeams-LiveShare', () => {
+  describe('initialize', () => {
+    it('should throw error due to package not being found', async () => {
+      await expect(liveShare.initialize()).rejects.toThrowError(
+        'Unable to initialize Live Share client. Ensure that your project includes "@microsoft/live-share"',
+      );
+    });
+  });
+
+  describe('joinContainer', () => {
```

Comment:
You need some unit tests that verify the "correct" path works (the right client function is called)

Reply:
Same as other comment

Reply:
Removed namespace per discussion